packages/cf.js/src/provider.ts

```diff
@@ -1,85 +1,82 @@
-import {
-  INodeProvider,
-  NodeMessage,
-  NodeMessageType,
-  NodeQueryData,
-  QueryType
-} from "@counterfactual/node-provider";
-
 import cuid from "cuid";
 
 import { AppInstance } from "./app-instance";
+import { INodeProvider, Node } from "./types";
 
-export enum CounterfactualEventType {
-  INSTALL = "cf_install",
-  PROPOSE_INSTALL = "cf_proposeInstall",
-  REJECT_INSTALL = "cf_rejectInstall"
-}
+export import DappEventType = Node.EventName;
```

Comment:
This is a kind of weird style at first glance `export import` - is this the best way to achieve this?

Reply:
I know, surprised me too. But this is the only way to alias an enum in both type and values.

Reply:
https://github.com/Microsoft/TypeScript/issues/3832

Reply:
TIL